src/controller/python/chip/ChipCoreBluetoothMgr.py

```diff
@@ -106,6 +106,15 @@ def _VoidPtrToCBUUID(ptr, len):
 
     return ptr
 
+class BgLoopCondition:
```

Comment:
Bg is "Background" ?

We should abbreviating if it creates uncertainty and just say "Background". Or just remove "Bg" altogether if it's not carrying it's weight.

Reply:
Done